docs/operations/blocks-storage.md

```diff
@@ -11,6 +11,7 @@ The supported backends for the blocks storage are:
 
 * [Amazon S3](https://aws.amazon.com/s3)
 * [Google Cloud Storage](https://cloud.google.com/storage/)
+* [Microsoft Azure Storage](https://azure.microsoft.com/en-us/services/storage/)
```

Comment:
May you also reference it in the `docs/architecture.md` within the section "Blocks storage (experimental)", please?

Reply:
Yes, thank you for calling that out.